addons/im_livechat/static/src/views/discuss_channel_list/discuss_channel_list_view_controller.js

```diff
@@ -9,9 +9,6 @@ export class DiscussChannelListController extends ListController {
     }
 
     async openRecord(record) {
-        if (!record.data.is_member) {
-            return super.openRecord(record);
-        }
         this.actionService.doAction("mail.action_discuss", {
```

Comment:
considering to use a different action from here if we still need customizations (sidebar, panel)

but I'd argue that we should avoid customizations